ModelRuntime testing failed cause huggingface repo not found

### Self Checks

- [x] This is only for bug report, if you would like to ask a question, please head to [Discussions](https://github.com/langgenius/dify/discussions/categories/general).
- [x] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [x] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [x] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [x] Please do not modify this template :) and fill in all the required fields.

### Dify version

main

### Cloud or Self Hosted

Self Hosted (Source)

### Steps to reproduce

```
=========================== short test summary info ============================
FAILED api/tests/integration_tests/model_runtime/huggingface_hub/test_llm.py::test_hosted_inference_api_validate_credentials[setup_huggingface_mock0] - core.model_runtime.errors.validate.CredentialsValidateFailedError: 401 Client Error. (Request ID: Root=1-679ee6d5-7d9879f266d23da12f01f984;7a15fcea-c754-4d97-b725-f98de6280cb8)

Repository Not Found for url: https://huggingface.co/api/models/HuggingFaceH4/zephyr-7b-beta.
Please make sure you specified the correct `repo_id` and `repo_type`.
If you are trying to access a private or gated repo, make sure you are authenticated.
Invalid credentials in Authorization header
================= 1 failed, 109 passed, 10 warnings in 12.89s ==================
```

### ✔️ Expected Behavior

_No response_

### ❌ Actual Behavior

_No response_

--- a/api/tests/integration_tests/model_runtime/huggingface_hub/test_llm.py
+++ b/api/tests/integration_tests/model_runtime/huggingface_hub/test_llm.py
@@ -10,6 +10,7 @@ from core.model_runtime.model_providers.huggingface_hub.llm.llm import Huggingfa
 from tests.integration_tests.model_runtime.__mock.huggingface import setup_huggingface_mock
 
 
+@pytest.mark.skip
 @pytest.mark.parametrize("setup_huggingface_mock", [["none"]], indirect=True)
 def test_hosted_inference_api_validate_credentials(setup_huggingface_mock):
     model = HuggingfaceHubLargeLanguageModel()

